src/meta/mod.rs

```diff
@@ -56,25 +57,51 @@ impl Meta {
             println!("cannot access '{}': {}", path.display(), err);
             return Ok(meta);
         }
-
         let mut content = Vec::new();
+
+         match display {
+            Display::DisplayAll => {
+                let mut current_meta;
+                let mut parent_meta;
+
+                let parent_path = match path.parent() {
+                    None => PathBuf::from("/"),
+                    Some(path) => PathBuf::from(path),
+                };
+
+                current_meta = Self::from_path(&PathBuf::from(path))?;
+                current_meta.name.name = ".".to_string();
+
+                parent_meta = Self::from_path(&PathBuf::from(parent_path))?;
```

Comment:
The linter says that you can remove the `PathBuf::from(` for this line

Reply:
I changed match to if.

Which linter program do you use? I use emacs's racer, it don't give such suggestions.

Reply:
you can check the specific command into the .travis.yml file.